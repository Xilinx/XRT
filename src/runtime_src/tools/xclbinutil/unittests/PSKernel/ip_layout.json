{
    "ip_layout": {
        "m_count": "1",
        "m_ip_data": [
            {
                "m_type": "IP_KERNEL",
                "m_int_enable": "1",
                "m_interrupt_id": "0",
                "m_ip_control": "AP_CTRL_CHAIN",
                "m_base_address": "0x80000",
                "m_name": "vadd:vadd_1"
            }
        ]
    }
}
