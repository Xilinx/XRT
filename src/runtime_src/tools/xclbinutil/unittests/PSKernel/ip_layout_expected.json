{
    "ip_layout": {
        "m_count": "2",
        "m_ip_data": [
            {
                "m_type": "IP_KERNEL",
                "m_int_enable": "1",
                "m_interrupt_id": "0",
                "m_ip_control": "AP_CTRL_CHAIN",
                "m_base_address": "0x80000",
                "m_name": "vadd:vadd_1"
            },
            {
                "m_type": "IP_PS_KERNEL",
                "properties": "0x0",
                "m_base_address": "not_used",
                "m_name": "kernel0:scu_0"
            }
        ]
    }
}
